components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodCLIService.kt

```diff
@@ -65,7 +69,11 @@ class GitpodCLIService : RestService() {
             val file = parseFilePath(fileStr) ?: return "invalid file"
             val shouldWait = getBooleanParameter("wait", urlDecoder)
             return withClient(request, context) {
-                CommandLineProcessor.doOpenFileOrProject(file, shouldWait).future.get()
+                GlobalScope.launch {
+                    withContext(Dispatchers.IO) {
+                        CommandLineProcessor.doOpenFileOrProject(file, shouldWait).future.get()
+                    }
+                }
```

Comment:
ℹ️ This change is because this became a suspend function in 2022.3.
<img width="875" alt="image" src="https://user-images.githubusercontent.com/418083/192543943-95a67736-db0d-4def-9005-a986c6c90397.png">